Recursive installation fails

While using httpmockserver as an indirect dependency, the installation fails during package building using cpack. This issue is generally with the installation not only cpack.

```
src/CMakeLists.txt
install(TARGETS ${MyTarget}                                                            
        RUNTIME                                                                  
        DESTINATION ${CMAKE_INSTALL_BINDIR}/${MyTarget})     
```

```
CPack: Create package using DEB
CPack: Install projects
CPack: - Run preinstall target for: myproject
CPack: - Install project: myproject []
CMake Error at /myproject/build/external/elasticlient/external/httpmockserver/cmake_install.cmake:50 (file):
  file INSTALL cannot find "/myproject/build/libhttpmockserver.pc": No such file
  or directory.
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -63,5 +63,5 @@ install(DIRECTORY ${HTTPMOCKSERVER_INCLUDE_DIRS}/httpmockserver
 # set pkg-config file
 set(prefix "${CMAKE_INSTALL_PREFIX}")
 configure_file("libhttpmockserver.pc.in" "libhttpmockserver.pc" @ONLY)
-install(FILES ${CMAKE_BINARY_DIR}/libhttpmockserver.pc DESTINATION lib/pkgconfig)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libhttpmockserver.pc DESTINATION lib/pkgconfig)
 

